roles/foreman/tasks/main.yaml

```diff
@@ -87,6 +87,20 @@
       DYNFLOW_REDIS_URL: "redis://localhost:6379/6"
       REDIS_PROVIDER: "DYNFLOW_REDIS_URL"
     command: "/usr/libexec/foreman/sidekiq-selinux -e production -r /usr/share/foreman/extras/dynflow-sidekiq.rb -C /etc/foreman/dynflow/%i.yml"
+    quadlet_options:
+      - |
+        [Install]
+        WantedBy=default.target
+
+- name: Create Dynflow Container instances
+  ansible.builtin.file:
+    state: link
+    src: "/etc/containers/systemd/dynflow-sidekiq@.container"
+    dest: "/etc/containers/systemd/dynflow-sidekiq@{{ item }}.container"
+  loop:
+    - orchestrator
+    - worker
+    - worker-hosts-queue
```

Comment:
Tests for each of these? 

Does the container startup command derive what persona to be based on the name of the service?

Reply:
Test that `/etc/containers/systemd/dynflow-sidekiq@{{ item }}.container` exists? Can add.